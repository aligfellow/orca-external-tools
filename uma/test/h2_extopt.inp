! ExtOpt sp

%pal
nprocs 4
end

%method
ProgExt "../../../umaclient.sh"
end

* xyz 0 1
H 0.0 0.0 0.0
H 0.0 0.0 0.9
*
