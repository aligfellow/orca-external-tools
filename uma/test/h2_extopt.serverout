UMASERVER_PID: 2653234
INFO:waitress:Serving on http://127.0.0.1:49463
