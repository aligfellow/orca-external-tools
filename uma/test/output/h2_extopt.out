
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ,####,    ,#,          
            ##       ##  ##   ,#'  ##            #'  '#       #'       ,# #          
            ##       ##  #######   ##           ,######,      #####,      #          
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,     #,   #      #          
             '#######'   ##     ##  '#######'  #'      '#     '####' #    #          



                #########################################################
                #                        -***-                          #
                #          Department of theory and spectroscopy        #
                #                                                       #
                #                      Frank Neese                      #
                #                                                       #
                #     Directorship, Architecture, Infrastructure        #
                #                    SHARK, DRIVERS                     #
                #        Core code/Algorithms in most modules           #
                #                                                       #
                #        Max Planck Institute fuer Kohlenforschung      #
                #                Kaiser Wilhelm Platz 1                 #
                #                 D-45470 Muelheim/Ruhr                 #
                #                      Germany                          #
                #                                                       #
                #                  All rights reserved                  #
                #                        -***-                          #
                #########################################################


                         Program Version 6.1.0  -  RELEASE   -
                                (GIT: $679e74b$)
                          ($2025-06-10 18:02:51 +0200$)


 With contributions from (in alphabetic order):
[Max-Planck-Institut fuer Kohlenforschung]                               
  Daniel Aravena         : Magnetic Suceptibility                        
  Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
  Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum      
  Ute Becker             : All parallelization in ORCA, NUMFREQ, NUMCALC 
  Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD           
  Dmytro Bykov           : pre 5.0 version of the SCF Hessian            
  Marcos Casanova-PÃ¡ez   : Triplet and SCS-CIS(D). UHF-(DLPNO)-IP/EA/STEOM-CCSD. UHF-CVS-IP/STEOM-CCSD   
  Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE 
  Pauline Colinet        : FMM embedding                                 
  Dipayan Datta          : RHF DLPNO-CCSD density                        
  Achintya Kumar Dutta   : EOM-CC, STEOM-CC                              
  Nicolas Foglia         : Exact transition moments, OPA infrastructure, MCD improvements    
  Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI      
  Miquel Garcia-Rates    : C-PCM and meta-GGA Hessian, CCSD/C-PCM, Gaussian charge scheme 
  Tiago L. C. Gouveia    : GS-ROHF, GS-ROCIS                             
  Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization        
  Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods 
  Ingolf Harden          : AUTO-CI MPn and infrastructure                
  Benjamin Helmich-Paris : MC-RPA, TRAH-(SCF,CASSCF), AVAS, COSX integrals, SCF dyn. polar., MC-PDFT, srDFT 
  Lee Huntington         : MR-EOM, pCC                                   
  Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM     
  Riya Kayal             : Wick's Theorem for AUTO-CI, AUTO-CI UHF-CCSDT 
  Emily Kempfer          : AUTO-CI RHF CISDT and CCSDT, approximate NEVPT4
  Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K, improved NEVPT2 
  Axel Koslowski         : Symmetry handling                             
  Simone Kossmann        : meta-GGA functionals, TD-DFT gradient, OOMP2, (MP2 Hessian; deprecated post 5.0) 
  Lucas Lang             : DCDCAS, Hyperfine gauge corrections, ICE-SOC+SSC
  Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC        
  Spencer Leger          : CASSCF response                               
  Dagmar Lenk            : GEPOL surface, SMD, ORCA-2-JSON               
  Dimitrios Liakos       : Extrapolation schemes; Compound Job, Property file
  Dimitrios Manganas     : Further ROCIS development; embedding schemes. LFT, Crystal Embedding 
  Dimitrios Pantazis     : SARC Basis sets                               
  Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients 
  Taras Petrenko         : pre 6.0 DFT Hessian and TD-DFT gradient,  ECA, NRVS 
  Petra Pikulova         : Analytic Raman intensities                    
  Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient                 
  Shashank Vittal Rao    : ES-AILFT, MagRelax                            
  Christoph Reimann      : Effective Core Potentials                     
  Marius Retegan         : Local ZFS, SOC                                
  Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples 
  Michael Roemelt        : Original ROCIS implementation, recursive CI coupling coefficients
  Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density      
  Barbara Sandhoefer     : DKH picture change effects                    
  Yorick L. A. Schmerwitz: GMF and freeze-and-release deltaSCF, NEB S-IDPP initial path                    
  Kantharuban Sivalingam : CASSCF convergence/infrastructure, NEVPT2, NEVPT3, NEVPT4(SD), FIC-MRCI and CEPA variants
  Bernardo de Souza      : ESD, SOC TD-DFT                               
  Georgi L. Stoychev     : AutoAux, RI-MP2 NMR, DLPNO-MP2 response, X2C  
  Van Anh Tran           : RI-MP2 g-tensors                              
  Willem Van den Heuvel  : Paramagnetic NMR                              
  Zikuan Wang            : NOTCH, Electric field optimization            
  Frank Wennmohs         : Technical directorship and infrastructure     
  Hang Xu                : AUTO-CI-Response properties                   
                                                                         
[FACCTs GmbH]                                                            
  Markus Bursch, Nicolas Foglia, Miquel Garcia-Rates, Ingolf Harden, Hagen Neugebauer, Anastasios Papadopoulos, 
  Christoph Riplinger, Bernardo de Souza, Georgi L. Stoychev                                                    
                                                                 
  APM, various basis sets, CI-OPT, improved COSX, DLPNO-Multilevel,           
  DOCKER, DRACO, updates on ESD, Fragmentator, GOAT, IRC, LR-CPCM, L-BFGS, MBIS, meta-GGA TD-DFT gradient, ML-optimized integration grids, 
  MM, NACMEs, nearIR, NEB, NEB-TS, NL-DFT gradient (VV10), 2- and 3-layer-ONIOM, interface openCOSMO-RS, QMMM,   
  Crystal-QMMM, RESP, rigid body optimization, SF, symmetry and pop. for TD-DFT, various functionals, SOLVATOR                                   
                                                                         
[Other institutions]                                                     
  V. Asgeirsson          : NEB                                           
  Christoph Bannwarth    : sTDA-DFT, sTD-DFT, PBEh-3c, B97-3c, D3        
  Giovanni Bistoni       : ETS/NOCV, ADLD/ADEX, COVALED                  
  Martin Brehm           : Molecular dynamics                            
  Ronald Cardenas        : ETS/NOCV                                      
  Martina Colucci        : COVALED                                       
  Sebastian Ehlert       : rSCAN, r2SCAN, r2SCAN-3c, D4, dhf basis sets  
  Marvin Friede          : D4 for Fr, Ra, Ac-Lr                          
  Lars Goerigk           : TD-DFT with DH, B97 family of functionals     
  Stefan Grimme          : VdW corrections, initial TS optimization, DFT functionals, gCP, sTDA/sTD-DF 
  Waldemar Hujo          : DFT-NL                                        
  H. Jonsson             : NEB                                           
  Holger Kruse           : gCP                                           
  Marcel Mueller         : wB97X-3c, vDZP basis set                      
  Hagen Neugebauer       : wr2SCAN, Native XTB                           
  Gianluca Regni         : ADLD/ADEX                                     
  Tobias Risthaus        : pre 6.0 range-separated hybrid DFT and stability analysis
  Lukas Wittmann         : regularized MP2, r2SCAN double-hybrids, wr2SCAN        
                                                                         
We gratefully acknowledge several colleagues who have allowed us to      
interface, adapt or use parts of their codes:                            
  Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods 
  Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG                   
  Ulf Ekstrom                                   : XCFun DFT Library      
  Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
  Frank Weinhold                                : gennbo (NPA and NBO analysis)           
  Simon Mueller                                 : openCOSMO-RS                            
  Christopher J. Cramer and Donald G. Truhlar   : smd solvation model                     
  S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library                           
  Liviu Ungur et al                             : ANISO software                          


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 7.0.0
 For citations please refer to: https://libxc.gitlab.io

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices
   BLAS/LAPACK       :  OpenBLAS 0.3.29  USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell SINGLE_THREADED
        Core in use  :  Haswell
   Copyright (c) 2011-2014, The OpenBLAS Project


================================================================================

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================


Using external energy/gradient program given in input:
  "../../../umaclient.sh"


WARNING: four-center integrals in TRAH must be done with SHARK!
  ===> : Turning Shark on


================================================================================
                                       INPUT FILE
================================================================================
NAME = h2_extopt.inp
|  1> ! ExtOpt sp
|  2> 
|  3> %pal
|  4> nprocs 4
|  5> end
|  6> 
|  7> %method
|  8> ProgExt "../../../umaclient.sh"
|  9> end
| 10> 
| 11> * xyz 0 1
| 12> H 0.0 0.0 0.0
| 13> H 0.0 0.0 0.9
| 14> *
| 15> 
| 16>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  H      0.000000    0.000000    0.000000
  H      0.000000    0.000000    0.900000

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 H     1.0000    0     1.008    0.000000    0.000000    0.000000
   1 H     1.0000    0     1.008    0.000000    0.000000    1.700754

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     0.900000000000     0.00000000     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     1.700753520529     0.00000000     0.00000000

Calling ../../../umaclient.sh h2_extopt_EXT.extinp.tmp   ... 

Total time:   4.702 seconds

External program completed successfully in 4.746 sec

-------------------------------------------------   --------------------
FINAL SINGLE POINT ENERGY (From external program)        -1.148454794868
-------------------------------------------------   --------------------


--------------------------------
SUGGESTED CITATIONS FOR THIS RUN
--------------------------------

Below you find a list of papers that are relevant to this ORCA run
We neither can nor want to force you to cite these papers, but we appreciate if you do
You receive ORCA, which is the product of decades of hard work by many enthusiastic individuals, for free
The only thing we kindly ask in return is that you cite our papers,
We deeply appreciate it, if you show your appreciation for ORCA by not just citing the generic ORCA reference.

Please note that relegating all ORCA citations to the supporting information does *not* help us.
SI sections are not indexed - citations you put there will not count into any citation statistics
But we need these citations in order to attract the funding resources that allow us to do what we are doing

Therefore, if you are a happy ORCA user, please consider citing a few of the papers listed below in the main body of your paper

In addition to the list printed below, the program has created the file h2_extopt.bibtex that contains the list in bibtex format
You can import this file easily into all common literature databanks and citation aid programs


List of essential papers. We consider these as the minimum necessary citations

  1. Neese, F.
     Software update: the ORCA program system, version 6.0
     WIRES Comput. Molec. Sci. 2025 15(1), e70019
     doi.org/10.1002/wcms.7019

List of papers to cite with high priority. The work reported in these papers was absolutely
necessary for this run to complete.
Our perspective: the developers of density functionals and basis sets usually get cited in chemistry papers
Good! But without the algorithms to do something with them, the functionals or basis sets would not do anything.
Hence, in our opinion, the algorithm design and method developments papers are equally worthy of getting cited

  1. Neese, F.
     The SHARK Integral Generation and Digestion System
     J. Comp. Chem. 2022 44(3), 381
     doi.org/10.1002/jcc.26942

List of suggested additional citations. These are papers that are important in the 'surrounding' of 
of this run, or papers that preceded the highly important papers. If you like your results we are grateful for a citation.

  1. Neese, F.
     The ORCA program system
     WIRES Comput. Molec. Sci. 2012 2(1), 73-78
     doi.org/10.1002/wcms.81
  2. Neese, F.
     Software update: the ORCA program system, version 4.0
     WIRES Comput. Molec. Sci. 2018 8(1), 1-6
     doi.org/10.1002/wcms.1327
  3. Neese, F.; Wennmohs, F.; Becker, U.; Riplinger, C.
     The ORCA quantum chemistry program package
     J. Chem. Phys. 2020 152(22), 224108
     doi.org/10.1063/5.0004608
  4. Neese, F.
     Software update: The ORCA program systemâVersion 5.0
     WIRES Comput. Molec. Sci. 2022 12(1), e1606
     doi.org/10.1002/wcms.1606

List of optional additional citations

  1. Neese, F.
     Approximate second-order SCF convergence for spin unrestricted wavefunctions
     Chem. Phys. Lett. 2000 325(1-3), 93-98
     doi.org/10.1016/s0009-2614(00)00662-x

Timings for individual modules:

Sum of individual times          ...        4.749 sec (=   0.079 min)
External energy and gradient     ...        4.749 sec (=   0.079 min) 100.0 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 5 seconds 585 msec
